### ISSUE TITLE
Post.accessibility_caption always returning None

### ISSUE THREAD

**Issue by peborges02** [instaloader/instaloader#2174]
**Describe the bug**
[Post.accessibility_caption](https://instaloader.github.io/module/structures.html#instaloader.Post.accessibility_caption) always returning None using Python Module.

It always worked fine (at least two months ago, when I ran my script for the last time)

**To Reproduce**
- Import session login: `L.load_session_from_file("username")`
- fetch some profile: `profile = instaloader.Profile.from_username(L.context, "some_profile")`
- iterate over posts: `for post in profile.get_posts():`
- get the alt text (accessibility_caption): `alt_text = post.accessibility_caption`
- accessibility_caption is `None`. But this post actually has alt text through Instagram Web

**Expected behavior**
The actual value of post's Alt Text (`accessibility_caption` property)

**Error messages and tracebacks**
No error

**Instaloader version**
4.10.3

**Additional context**
Using Python3 Instaloader Module installed via `pip3`

**Comment by aandergr** [comment]
Indeed, for Posts returned by Profile.get_posts() the accessibility_caption is returned as None. It still works for Posts.from_shortcode().

Thanks for reporting.
